node-repository/src/main/java/com/yahoo/vespa/hosted/provision/persistence/ApplicationSerializer.java

```diff
@@ -141,8 +140,6 @@ private static void toSlime(Cluster cluster, Cursor clusterObject) {
         toSlime(cluster.maxResources(), clusterObject.setObject(maxResourcesKey));
         toSlime(cluster.groupSize(), clusterObject.setObject(groupSizeKey));
         clusterObject.setBool(requiredKey, cluster.required());
-        // TODO(olaa): Remove 'suggested' once API clients migrate to suggestion list
-        toSlime(cluster.suggested(), clusterObject.setObject(suggestedKey));
```

Comment:
Configservers upgrade one-by-one, so you have to keep writing the field until there are no readers left. Stop reading the field first. Once that change has rolled out you can stop writing it and remove it entirely.